Testscripts/Linux/ETHTOOL-HANDLER-HASH-LEVELS.sh

```diff
@@ -69,30 +69,36 @@ if ! GetSynthNetInterfaces; then
     exit 0
 fi
 
-# Check if kernel support network flow hashing options with ethtool
-sts=$(ethtool -n "${SYNTH_NET_INTERFACES[@]}" rx-flow-hash "$protocol" 2>&1)
-CheckResults "$sts"
+INTERFACE=""
```

Comment:
I don't understand why you assigned the value from ifc to INTERFACE in the for-loop. Can't you use 'ifc' directly in the loop?

Reply:
Basically INTERFACE is defined as global which is used in the other functions too.

Reply:
acked